src/test/ui/nll/relate_tys/trait-hrtb.stderr

```diff
@@ -0,0 +1,8 @@
+error: higher-ranked subtype error
```

Comment:
Aside (not to be fixed here): This message is rather ungreat from the POV of someone who is not a type theorist. ;)

Reply:
Dup https://github.com/rust-lang/rust/issues/57374 :)